File: tests/fixtures/code_samples/java/sample_040.java
// Sample 40: small utility.
package samples;

import java.util.List;

public final class Sample040 {
    private Sample040() {}

    public static int operation(List<Integer> xs) {
        int total = 40;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 40) %% 7919;
    }
}

File: tests/fixtures/code_samples/py/sample_040.py
"""Sample 40: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_40(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 40."""
    total = 40
    for x in xs:
        total += int(x)
    return total


def operation_40_pure(value: int) -> int:
    return (value * 40) % 7919

